DisplayOptionsPanel.c

```diff
@@ -67,16 +65,32 @@ static HandlerResult DisplayOptionsPanel_eventHandler(Panel* super, int ch) {
          break;
       case '-':
       case KEY_F(7):
+      case KEY_RIGHTCLICK:
```

Comment:
I object using right click as mentioned in [this comment](https://github.com/htop-dev/htop/pull/1731#issuecomment-3014042910). The right button to use should be `KEY_WHEELDOWN`.